components/layouts/RightBar.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div id="right-bar">
+    <v-navigation-drawer
+      v-model="isNavigation"
+      right
+      hide-overlay
+      permanent
+      color="#293039"
+      :width="'350px'"
+    >
+      <v-switch
+        class="ml-4"
+        v-model="isSelectAll"
+        inset
+        label="Select all sub keypath"
+        color="rgba(0, 153, 204, 1)"
+        dark
+      ></v-switch>
+      <v-text-field
+        class="pa-2"
+        v-model="search"
+        placeholder="input keypath ..."
+        dark
+        dense
+        hide-details
+        outlined
+        @keydown.stop="inputKeypath"
+      ></v-text-field>
+      <v-treeview
+        :items="layerTree"
+        dark
+        activatable
+        hoverable
+        item-key="keypath"
+        item-text="keypath"
+        color="rgba(0, 153, 204, 1)"
+        @update:active="changeFocus"
+        open-all
+        ref="treeview"
+        :search="search"
+      >
+        <template v-slot:prepend="{ item, open }">
+          <v-icon v-if="item.type == 'root'">
+            {{ open ? 'mdi-folder-open' : 'mdi-folder' }}
+          </v-icon>
+          <v-icon v-else-if="item.type == '4'"> mdi-layers </v-icon>
+          <v-icon v-else-if="item.type == 'gr'"> mdi-crop </v-icon>
+          <v-icon v-else-if="item.type == 'st'"> mdi-pencil </v-icon>
+          <v-icon v-else-if="item.type == 'fl'"> mdi-format-color-fill </v-icon>
+          <v-icon v-else> mdi-cancel </v-icon>
+        </template>
+        <template v-slot:label="{ item }">
+          {{ item.name }}
+        </template>
+      </v-treeview>
+    </v-navigation-drawer>
+  </div>
+</template>
+
+<script>
+module.exports = {
+  name: 'right-bar',
+  props: ['height'],
+  data() {
+    return {
+      isNavigation: true,
+      search: '',
+      openFolder: ['root'],
+    };
+  },
+  computed: {
+    ...Vuex.mapGetters(['layerTree', 'keypath', 'selectedAllKeypath']),
+    isSelectAll: {
+      get() {
+        return this.$store.getters.isSelectAll;
+      },
+      set() {
+        this.$store.commit('setIsSelectAll');
+      },
+    },
+  },
+  mounted() {
+    // Shortcut key function binding
+    document.addEventListener('keydown', function (e) {
+      if (e.ctrlKey && e.which == 76) {
+        // Hide and show layer list : Ctrl + L
```

Comment:
Is this working for you?
It seems that there is not a key icon as the .Gif/manual_7.gif shows, and it is not able to close right bar using "Ctrl + L"
Does the "Hide and show layer list" mean "Hide and show layer list panel (whole right bar, as.Gif/manual_7.gif shows) "?